docs/static/modules.asciidoc

```diff
@@ -0,0 +1,121 @@
+[[logstash-modules]]
+== Working with Logstash Modules
+
+Logstash modules provide a quick way to get started using Logstash. 
+
+Each module comes pre-packaged with configurations, Kibana dashboards, and
+other meta files that make it easier for you to set up the Elastic stack for
+specific use cases or data sources.
+
+[float]
+[[running-logstash-modules]]
+=== Running modules
+
+When you run a module, Logstash creates and loads the pipeline configurations
+required to read and parse the data. It also loads the index pattern,
```

Comment:
index pattern and field mappings will be directly on ES, should we also make that explicit?